src/benchq/resource_estimation/azure.py

```diff
@@ -0,0 +1,133 @@
+################################################################################
+# © Copyright 2022-2023 Zapata Computing Inc.
+################################################################################
+import os
+from collections import Counter
+from dataclasses import dataclass
+from typing import Dict, Optional
+import warnings
+
+from azure.quantum.qiskit import AzureQuantumProvider
+from orquestra.integrations.qiskit.conversions import export_to_qiskit
+from orquestra.quantum.circuits import Circuit
+from qiskit.tools.monitor import job_monitor
+
+from ..data_structures import BasicArchitectureModel, QuantumProgram
+
+
+@dataclass
+class AzureResourceInfo:
+    physical_qubit_count: int
+    logical_qubit_count: int
+    total_time: float
+    depth: int
+    distance: int
+    cycle_time: float
+    logical_error_rate: float
+    raw_data: dict
+
+
+def _azure_result_to_resource_info(job_results: dict) -> AzureResourceInfo:
```

Comment:
thought: does this require its own function? It's only called in one place.

Reply:
I expect it might be used elsewhere in future.
Plus, it's just a helper function, so code is easier to read if it's here.